internal/server/server.go

```diff
@@ -117,6 +120,21 @@ func (s *Server) handleRoot() http.HandlerFunc {
 	}
 }
 
+func (s *Server) handleHealthz() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		status := map[string]string{
+			"status": "ok",
+		}
+
+		bytes, err := json.Marshal(status)
+		if err != nil {
+			logAndError(w, http.StatusBadRequest, err, "error gathering status")
```

Comment:
@dskatz You need a `return` after this error.

Reply:
doh!